runner/src/mill/runner/MillCliConfig.scala

```diff
@@ -1,4 +1,4 @@
-package mill
+package mill.runner
```

Comment:
Moved this to `runner` since that's where it belongs, and to simplify the scaladoc